src/configuration/configuration.d.mts

```diff
@@ -314,9 +314,9 @@ export interface SettingConfig {
       choices: InterfaceToObject<typeof CONFIG.Dice.rollModes>;
     },
     // Note(LukeAbby): This override is necessary because the `initial` wasn't removing `null`.
-    CONFIG.Dice.RollMode | null | undefined,
-    CONFIG.Dice.RollMode,
-    CONFIG.Dice.RollMode
+    CONFIG.Dice.ConfiguredRollModes | null | undefined,
```

Comment:
Well with how `RollMode` is set up isn't the index signature branded? Is this change necessary?

Reply:
Yup.